.github/workflows/ci.yml

```
@@ -47,6 +34,11 @@ jobs:
         uses: tree-sitter/parser-test-action@v2
         with:
           test-rust: ${{runner.os == 'Linux'}}
+      - name: Check unchanged parser
+        shell: bash
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2046:warning:2:6: Quote this to prevent word splitting

(shellcheck)